app/view_models/bids_new_view_model.rb

```diff
@@ -0,0 +1,167 @@
+class BidsNewViewModel
```

Comment:
Naming things is so hard. Let's just give everything random UUIDs as names. Maybe we should just do `NewBidViewModel`?